proprietary/Configure-TI-CC2541-compatible-proprietary-PHY.md

```diff
@@ -0,0 +1,134 @@
+</div> 
+For easier figure out the frame, it is worth to disable the whitening and set some special data bit on the frame, like add 0x00/0xFF.
+<div align="left">
+  <img src="files/CM-Configure-TI-CC2541-compatible-proprietary-PHY/Frame.png">  
+</div> 
+
+## 3.2. Capture data on MS2692A
+To get a readable frame data on MS2692A, we need to configure the frequency, reference power level etc. accordingly.
+### 3.2.1 Use "Power vs Time" trace mode to get the TX pluse.
+<div align="left">
+  <img src="files/CM-Configure-TI-CC2541-compatible-proprietary-PHY/SA-01-pluse.png">  
+</div> 
+
+### 3.2.2 Change the "Start Time" to locate one pluse.
+<div align="left">
+  <img src="files/CM-Configure-TI-CC2541-compatible-proprietary-PHY/SA-02-location.png">  
+</div> 
+
+### 3.2.3 Zoom in.
+<div align="left">
+  <img src="files/CM-Configure-TI-CC2541-compatible-proprietary-PHY/SA-03-zoom-in.png">  
+</div> 
+
+### 3.2.4 Use "Frequency vs Time" trace mode, the whole frame will be show up.
+Time between Marker1 and Marker 2 is 52us. This should be the preamble time. 13bytes preamble, 13x8x0.5us=52us. As we can see MS2629A only detect 1 byte preamble and got 48us CW signal. Still not know the reason, but checking only TI BLE PHY, it got the same issue.
+<div align="left">
+  <img src="files/CM-Configure-TI-CC2541-compatible-proprietary-PHY/SA-04-frame.png">  
+</div> 
+
+### 3.2.5 Read the frame data.
+Once the Preamble can be located, the frame data should be readable, we just try to sample the data bit for each 0.5us interval.
+<div align="left">
+  <img src="files/CM-Configure-TI-CC2541-compatible-proprietary-PHY/SA-05-syncwords.png">  
+</div> 
+
+# 4. EFR32xG22 Project
+After checking the frame data, now we can create and configure on our EFR32xG22 project, here we create "Flex (RAIL) - RAILtest" example on SSv5.
+<div align="left">
+  <img src="files/CM-Configure-TI-CC2541-compatible-proprietary-PHY/railtest.png">  
+</div> 
+
+## 4.1. Customize the PHY
+Double click "radio_settings.radioconf", select one PHY, then check on "Customized", change frequency to 2466MHz.
+<div align="left">
+  <img src="files/CM-Configure-TI-CC2541-compatible-proprietary-PHY/customized.png">  
+</div> 
+Configure according to arget PHY. 2Mbps data rate, 500KHz deviation. Configure flexible payload length, use CCITT_16(0x1021) polynomial and 0xFFFF CRC seed. Enable whitening. The Preamble set 8 bit here.
+<div align="left">
+  <img src="files/CM-Configure-TI-CC2541-compatible-proprietary-PHY/packet.png">  
+</div> 
+Uncheck "Number of Symbols in Timing Window" for disable preamble.
+<div align="left">
+  <img src="files/CM-Configure-TI-CC2541-compatible-proprietary-PHY/timing.png">  
+</div> 
+
+
+Left the other parameter as default, the PHY is done. Save and build, now then software is ready.  
+For more infromation on radio configuratore, refer to [AN1253](https://www.silabs.com/documents/public/application-notes/an1253-efr32-radio-configurator-guide-for-ssv5.pdf). 
```

Comment:
For more information on radio configurator.

Reply:
fixed, typo